ad/movement/trusts.md

```diff
@@ -28,6 +28,16 @@ Simply establishing a trust relationship does not automatically grant access to
 A trust relationship allows users in one domain to **authenticate** to the other domain's resources, but it does not automatically grant access to them. Access to resources is controlled by permissions, which must be granted explicitly to the user in order for them to access the resources.
 {% endhint %}
 
+### Global Catalog
+The global catalog is a partial copy of all objects in an Active Directory forest, meaning that some object properties (but not all) are contained within it. This data is replicated among all domain controllers marked as global catalogs for the forest. One point of the global catalog is to allow for object searching and deconfliction quickly without the need for referrals to other domains (more information here).
```

Comment:
1. `(more information here` > what does `here` refer to?
2. Let's change a few things to make it more smooth and easy to read
   1. `One point of the global catalog` --> `One of the Global Catalog's purposes`
   2. `to allow for object searching and deconfliction quickly` --> `to facilitate quick object searching and conflict resolution`
   3. `without the need for referrals to other domains` --> `without the necessity of referring to other domains`

Reply:
Completed!